scene/gui/graph_edit.cpp

```diff
@@ -71,6 +72,23 @@ Error GraphEdit::connect_node(const StringName &p_from, int p_from_port, const S
 	return OK;
 }
 
+void GraphEdit::add_zoom_level(const float p_zoom_level) {
```

Comment:
I would probably use indices to refer to zoom levels, not a float.. an API like
void add_zoom_level(float,Theme)
int get_zoom_level_count()
void remove_zoom_level(int)

And it's kind of your responsibility to add them in order, so the - and + buttons work properly